integration-cli/docker_cli_cp_test.go

```diff
@@ -472,7 +472,7 @@ func TestCpVolumePath(t *testing.T) {
 	}
 
 	cleanedContainerID := strings.TrimSpace(out)
-	defer deleteContainer(cleanedContainerID)
+	defer dockerCmd(t, "rm", "-fv", cleanedContainerID)
```

Comment:
but `-f` is always doing a sigkill, and isn't `deleteContainer` doing this exact thing?